pages/getting-started/first-steps-with-docker.mdx

```diff
@@ -0,0 +1,491 @@
+
+If you are working with the `memgraph-platform` image, you should pass
+configuration options with environment variables.
+
+For example, if you want to limit memory usage for the whole instance to 50 MiB
+and set the log level to `TRACE`, pass the configuration like this:
+
+```
+docker run -p 7687:7687 -p 3000:3000 -p 7444:7444 -e MEMGRAPH="--memory-limit=50 --log-level=TRACE" memgraph/memgraph-platform
+```
+
+When you are working with `memgraph` or `memgraph-mage` images, you should pass
+configuration options as arguments.
+
+For example, if you want to limit memory usage for the whole instance to 50 MiB
+and set the log level to `TRACE`, pass the configuration argument like this:
+
+```
+docker run -p 7687:7687 memgraph/memgraph --memory-limit=50 --log-level=TRACE
+```
+
+#### Specify volumes
+
+Specifying a volume syncs the specified directory inside the Docker container as
+a local directory and serves for durability. The `-v` flag is followed by the
+name of the local directory separated from the path of the volume in the
+container by a semicolon:
+
+```
+-v volume_name:volume_path
+```
+
+Named volumes handle data permissions so there shouldn't be any issue with data
+permissions. 
+
+Useful volumes you can specify while running Memgraph are:
+
+- `-v mg_lib:/var/lib/memgraph` - directory containing data, enables data
+  persistency
+- `-v mg_log:/var/log/memgraph` - directory containing log files
+- `-v mg_etc:/etc/memgraph` - directory containing the configuration file
+
+The exact location of the local directories depends on your specific setup.
+
+The configuration file can usually be found at
+`/var/lib/docker/volumes/mg_etc/_data/memgraph.conf` but you can also copy the
+file from the Docker container, modify it and copy it back into the container.
+
+The logs will be saved to the `mg_log` volume, and the directories can usually be
+found in `/var/lib/docker/volumes/`, but you can also view them in the Memgraph
+Lab 2.0 (or newer) by publishing the port `7444`.
+
+#### Specify bind mounts
+
+Bind mounts are local directories or files that can be modified by other
+processes other than Docker. Any changes made to these directories or files
+locally will be reflected inside the Docker container and vice-versa. Also, a
+bind mount will overwrite the content of the Docker container.
+
+For example, if I have a `Data` directory on my `C:` disk, and I want to access
+it from inside the container at `/usr/lib/memgraph/data`, I would run Docker
+with the following `-v` flag.
+
+```
+docker run -p 7687:7687 -p 7444:7444 -p 3000:3000  -v "C:/data":/usr/lib/memgraph/data memgraph/memgraph-platform
+```
+
+You can use bind mounts to transfer durability files such as snapshot or WAL
+files inside the container to restore data, or CSV files you will use to import
+data with `CSV LOAD` clause. 
+
+Bind mounts do not handle data permissions which could cause issues with
+permissions. 
+
+### Stop container
```

Comment:
maybe you can add a callout that stopping isn't deleting and link to that last chapter